cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -619,10 +619,20 @@ For services, you can retrieve:
 
 For workflows, you can retrieve:
 
-- the information about the specific run currently running::
```

Comment:
I think since we have a single bullet now, *run* should change to *node*. What do you think?